.azure/templates/build-test.yml

```diff
@@ -88,7 +88,7 @@ steps:
       set -e -x
       sudo apt-get install libacl1-dev libncurses5-dev pkg-config
       git clone --depth 1 \
-                --branch "${ICEORYX_BRANCH:-v1.0.1}" \
+                --branch "${ICEORYX_BRANCH:-v1.90.0}" \
```

Comment:
This tag will probably not work with https://github.com/eclipse-cyclonedds/cyclonedds/pull/1175/files since the storage is not used anymore...

I.e. it will not work with iceoryx master (2.0) and 1.x at the same time.